CODEOWNERS

```diff
@@ -61,7 +61,7 @@ deps.bzl            @digital-asset/daml
 /web-ide/           @digital-asset/daml
 
 # Misc
-/docs/              @digital-asset/daml
+/docs/              @bethaitman
```

Comment:
This means that you're the only owner of the docs. Is that what you want?

Reply:
I guess not? But I don't know who else I should add - @bame-da?

Reply:
There's another PR in flight which will have `@daml/empty` as the default. Just add yourself to that once it has landed.